pkg/ingester/metrics.go

```diff
@@ -187,8 +189,16 @@ func newIngesterMetrics(r prometheus.Registerer, createMetricsConflictingWithTSD
 			Name: "cortex_ingester_memory_chunks",
 			Help: "The total number of chunks in memory.",
 		}),
+		seriesEnqueuedForFlush: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_ingester_enqueued_series_total",
```

Comment:
I've the feeling the metric name is too much generic. Have you considered something like `cortex_ingester_series_flushing_enqueued_total`?

Reply:
Going with `cortex_ingester_flushing_enqueued_series_total` to keep `series` (unit of the metric) close to the end. Same for dequeued.